cloned from svn://gcc.gnu.org/svn/gcc/trunk/libstdc++-v3/python

Applied fix for this bug
https://bugs.launchpad.net/ubuntu/+source/gdb/+bug/1258642
